src/common/constants.ts

```diff
@@ -114,3 +114,5 @@ export type AllModals = keyof typeof reverseMap;
 
 export const withApiServer = (apiServer: string) => (path?: string) =>
   path ? apiServer + path : apiServer;
+
+export const POX_ADDRESS = 'SP000000000000000000002Q6VF78';
```

Comment:
Same question of how this is used - and it doesn't ever seem to be converted to a testnet addr?

Reply:
Unrelated nit: the appDetails: icon url is not great at bigger widths. In the 'choose account' view it looks a little blurry. Would recommend bumping it to ~256px or so, but it also seems like the root image itself is not quite big enough.